hw/ip_templates/pwrmgr/pwrmgr_pkg.core.tpl

```diff
@@ -11,6 +11,9 @@ filesets:
   files_rtl:
     depend:
       - ${instance_vlnv("lowrisc:ip:pwrmgr_reg")}
+% if wait_for_external_reset:
+      - lowrisc:ip:rom_ctrl_pkg
```

Comment:
What triggers this dependency?

Reply:
I was thinking the `boot_status_o` output had some dependency, but that was wrong. I removed that dependency.

Reply:
It's `rom_ctrl_pkg::pwrmgr_data_t` in the boot status.

Reply:
So this dependency is still needed, right?